tests/shardddl3/conf/single-source-optimistic-2.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: test
+task-mode: all
+is-sharding: true
+shard-mode: "optimistic"
+meta-schema: "dm_meta"
+timezone: "Asia/Shanghai"
+ignore-checking-items: ["table_schema"]
```

Comment:
Didn't know what blocks MySQL 8.0 passing it

Reply:
unnecessary, I just copied from other files. removed in https://github.com/pingcap/dm/pull/1171/commits/e93433fb537f9846ee0aee6c0645b5601fdddba6.